teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -6,10 +6,13 @@
    "source": [
```

Comment:
<blockquote>Simple examples for <strong>the </strong>PubChem API</blockquote><blockquote>Before query<strong>ing</strong> PubChem for similar compounds, we provide some simple examples to show how to use <strong>the</strong> PubChem API. For more detail about <strong>the </strong>PubChem API, see <a href="https://pubchemdocs.ncbi.nlm.nih.gov/pug-rest-tutorial" target="_blank">PUG-REST tutorial</a>.</blockquote><p> </p><p>Maybe add a short comment on how PubChem calculates ligand similarity?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>added the ligand similarity thing below</p>